README.md

```diff
@@ -2,15 +2,15 @@
 
 ```ruby
 class Document
-   attr_writer :content 
+   attr_accessor :content, :title
 
-  def initialise
+  def initialise content="" title=""
     @content = content
     @title = title
   end
 
   def to_s
-    "#{title}: #{content}"
+    "#{@title}: #{@content}"
```

Comment:
`attr_accessor` will create a `title` and `content` method so you wouldn't have to add the `@`. Because we are not using the constructor arguments and assigning `content` and `title` using the `attr_accessor` methods we could completely remove our initialize method.